docker-compose.yml

```diff
@@ -32,34 +32,37 @@ services:
       - kafka
 
   postgresql:
-    image: docker.io/bitnami/postgresql:14
+    image: postgres:16-alpine
     environment:
       - POSTGRES_PASSWORD=okteto
       - POSTGRES_USER=okteto
       - POSTGRES_DB=votes
     ports:
       - 5432
     volumes:
-      - /var/lib/postgresql
+      - postgresql_data:/var/lib/postgresql/data
 
   zookeeper:
```

Comment:
modern kafka doesn't need zookeper, check out https://github.com/okteto/voting-app/blob/main/docker-compose.yml for reference